crates/agglayer-storage/src/stores/state/tests.rs

```diff
@@ -233,6 +233,43 @@ fn can_read(network_id: NetworkId, store: StateStore) {
     ));
 }
 
+#[test]
+fn import_native_tokens() {
+    let certificates: Vec<Certificate> = ["cert_h0.json", "cert_h1.json", "cert_h2.json"]
+        .iter()
+        .map(|p| data::load_certificate(p))
+        .collect();
+
+    let mut lns = LocalNetworkStateData::default();
+
+    for (idx, certificate) in certificates.iter().enumerate() {
+        info!(
```

Comment:
Do you have logs enabled in the test?

Reply:
We're using `test_log` when needed.

Reply:
@Freyskeyd It is not fortunate to manually need to change every test we need the output from. For example if the CI build fails we can not see what happened. We should consider using `println` in the tests consistently.